src/data/general-cs-quiz.ts

```diff
@@ -1,4 +1,27 @@
 const generalCSQuiz = [
+  {
+    Question:
+      "Which sorting algorithm has an average and worst-case time complexity of O(nlogn)?",
+    Answer: "Quick sort",
+    Distractor1: "Bubble sort",
+    Distractor2: "Insertion sort",
+    Distractor3: "Selection sort",
+    Explanation:
+      "Quick sort has an average and worst-case time complexity of O(n log n), making it efficient for sorting large datasets.",
+    Link: "https://www.freecodecamp.org/news/sorting-algorithms-explained-with-examples-in-python-java-and-c/#classification-of-a-sorting-algorithm"
+  },
+  {
+    Question:
+      " In a graph, which data structure is best suited for finding the shortest path between two vertices?",
+    Answer: ". Priority Queue",
+    Distractor1: "Stack",
+    Distractor2: "Queue",
+    Distractor3: "Heap",
+    Explanation:
+      "Dijkstra's algorithm, which is used to find the shortest path in a graph, typically employs a            priority queue to efficiently select the next vertex with the smallest distance.",
```

Comment:
remove unnecessary spaces 

```suggestion
      "Dijkstra's algorithm, which is used to find the shortest path in a graph, typically employs a priority queue to efficiently select the next vertex with the smallest distance.",
```